src/org/anjocaido/groupmanager/utils/Tasks.java

```diff
@@ -63,19 +63,18 @@ public static String getStackTraceAsString(Exception exception) {
 
 	public static void copy(InputStream src, File dst) throws IOException {
 
-		InputStream in = src;
 		OutputStream out = new FileOutputStream(dst);
 
 		// Transfer bytes from in to out
 		byte[] buf = new byte[1024];
 		int len;
-		while ((len = in.read(buf)) > 0) {
+		while ((len = src.read(buf)) > 0) {

```

Comment:
Check this works as we don't want to be closing a stream that was passed to us if its going to be used after.

Reply:
In order to avoid causing issues (and because I don't know how this change can break anything), I will revert this